tests/test_embed_kernel.py

```diff
@@ -109,7 +109,10 @@ def test_embed_kernel_basic():
     with setup_kernel(cmd) as client:
         # oinfo a (int)
         client.inspect("a")
-        msg = client.get_shell_msg(timeout=TIMEOUT)
+        while True:
+            msg = client.get_shell_msg(timeout=TIMEOUT)
+            if msg["msg_type"] == "inspect_reply":
+                break
```

Comment:
does this have risk of hanging forever ?

Reply:
I removed it, it seems to be fine now.